src/common/com/intellij/plugins/haxe/ide/generation/GetterSetterMethodBuilder.java

```diff
@@ -20,10 +22,6 @@
 import com.intellij.plugins.haxe.model.HaxeFieldModel;
 import com.intellij.plugins.haxe.util.HaxePresentableUtil;
 
-/**
- * @author: Fedor.Korotkov
- * @author: Ilya Malanin
```

Comment:
On the other hand, it is kind of nice to know who originally wrote the a class, since they might be able to give you some insight.  These messages are no big deal either way and I know it's annoying to some folks when they show up in JavaDoc.  That should be impetus to put better documentation on every class created.